src/app/embed/adapters/merge.adapter.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable} from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { ReplaySubject } from 'rxjs';
+import { QSDAdapter } from './qsd.adapter'
+import { FeedAdapter } from './feed.adapter'
+import { AdapterProperties, hasMinimumParams, DataAdapter, getMergedValues } from './adapter.properties'
+
+const NO_EMIT_YET = Symbol();
+
+@Injectable()
+export class MergeAdapter {
+  private adapters: DataAdapter[]
+  constructor(
+    private FeedAdapter: FeedAdapter,
+    private QSDAdapter:  QSDAdapter
+	) { 
+    this.adapters = [this.QSDAdapter, this.FeedAdapter]
+  }
+
+  getProperties(params): Observable<AdapterProperties> {
+    let chosenAdapters: Observable<AdapterProperties>[];
+
+    chosenAdapters = this.adapters.
+      map(obs => obs.getProperties(params)).
+      map(obs => obs.startWith(NO_EMIT_YET))
+
+    return Observable.combineLatest(...chosenAdapters).map(sources => {
+      let data = [];
+      for (let source of sources) {
+        if (source === NO_EMIT_YET) { break; }
+        data.push(source);
+      }
```

Comment:
Looking through this now, and given the way that you seem to want to think about other adapters that you may introduce, it might make sense to call this adapter PriorityAdapter and have a MergeAdapter that does most of the same stuff this one does but without this break, essentially.